2023/README.md

```diff
@@ -1,20 +1,284 @@
+The process we went through to arrive at our priority list for Interop 2023 is available in the [planning process](planning-process.md) document.
+
+Similar to [Interop 2022](https://wpt.fyi/interop-2022), the tests we selected will be continuously run on automated testing infrastructure. The test pass rates for each browser rendering engine will be displayed on the [Interop 2023 Dashboard](https://wpt.fyi/interop-2023) — displaying the percentage of passing tests in each chosen area, and an overall total score of tests that pass in every browser. The dashboard will also display scores for group progress on particular Investigation Efforts selected for the Interop team to work on throughout the year.
+
+Interop 2023 is tightly focused on technology that is already specified in web standards.
 
-Interop 2023 is tightly focused on technology that is already specified in web standards. This is not a place to invent new web technology, nor solve complex problems that need web standards. That work should happen in the standards bodies.
 
 ## Dashboard
 
+Our dashboard lives at [wpt.fyi/interop-2023](https://wpt.fyi/interop-2023). It computes an interop score as the percentage of tests in WPT that pass in all three browser engines.
 
 
 ## Focus Areas
 
+The folloing features were selected for Interop 2023. Descriptions are taken from issue proposal submissions and MDN documentation.
+
+
+### Border Image in CSS
+
+Authors can specify an image to be used in place of the border-style. In this case, the border’s design is taken from the sides and corners of an image specified ([CSS `border-image`](https://github.com/web-platform-tests/interop/issues/146)).
+
+[MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/border-image)
+
+[Spec](https://drafts.csswg.org/css-backgrounds-3/#the-border-image)
+
+[Tests](https://wpt.fyi/results/css/css-backgrounds?label=experimental&label=master&product=chrome&product=firefox&product=safari&aligned&view=interop&q=label%3Ainterop-2023-cssborderimage)
+
+
+### Color Spaces and Functions in CSS
+
+[MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value)
+
+[Spec](https://drafts.csswg.org/css-color-4/)
+
+[Tests](https://wpt.fyi/results/css?label=experimental&label=master&product=chrome&product=firefox&product=safari&aligned&view=interop&q=label%3Ainterop-2022-color%20or%20label%3Ainterop-2023-color)
+
+
+### Container Queries in CSS
+
+CSS containment provides a way to isolate parts of a page and declare to the browser these parts are independent from the rest of the page in terms of styles and layout. ([Container Queries](https://github.com/web-platform-tests/interop/issues/133)).
```

Comment:
This seems to be more a description of containment than container queries, and we have containment following this.

Reply:
Yes, I think this paragraph is taken from the top of the [CSS Container Queries MDN page](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Container_Queries). Since containment is covered later, it's probably better to use a paragraph later from the MDN page.

Reply:
I submitted an updated text